### PR TITLE
Add session in for DEV3

### PR DESCRIPTION
**Pull request by davetheunissen**
Session info and blurb

### CODE REVIEW

File: GWABAklWebApplication/akldata.json [hunk 1]
```diff
@@ -147,13 +147,12 @@
...
           "Speaker": {
             "Name": "Dave Theunissen",
-            "Organization": "Sr. Software Engineer / Cloud & DevOps Consultant",
+            "Organization": "Cloud & DevOps Consultant @ Fidelity Life Assurance Company",
             "Description": "David is a Software Engineer with a focus on leading cloud adoption and DevOps practices. He's been developing systems on the Microsoft .NET stack for almost 10 years, and is passionate about software engineering, DevOps practices and Azure related technologies. His focus over the past few years has been in cloud and microservice architecture, development and continuous delivery. He is an active member of the wider the development community, sharing and talking about Azure technologies through his blog, social media and at various meetup groups.",
-            "ImageUrl": "/images/DaveTheunissen.jpg",
```
Comment (PlagueHO) [inline GWABAklWebApplication/akldata.json]: Why remove your image?

Comment (davetheunissen) [reply GWABAklWebApplication/akldata.json]: sorted

**Comment by PlagueHO** [comment]
:lgtm:

**Review by PlagueHO — APPROVED** [review]
:lgtm:

**Pull request merged**